components/emoji_picker/modules/emoji_selector.vue

```diff
@@ -0,0 +1,87 @@
+<template>
+  <div class="d-emoji-picker__selector">
+    <!--    Emoji Selector -->
+    <div
+      v-for="emoji in EMOJI_LIST"
+      :key="emoji"
+      class="d-emoji-picker__selector--emoji-list"
+    >
+      <p
+        class="d-emoji-picker__selector--emoji-list__tab"
+      >
+        {{ emojiFilter ? 'Search results' : emoji.tab }}
+      </p>
+
+      <div>
+        <dt-emoji
+          v-for="emoji in filteredEmojiList"
+          :key="emoji.code"
+          class="d-emoji-picker__selector--emoji-list__emoji"
+          :code="emoji.code"
+          @mouseover="$emit('emoji-data', emoji)"
+          @mouseleave="$emit('emoji-data', null)"
+          @click="$emit('selected-emoji', emoji)"
+        />
+      </div>
+    </div>
+  </div>
+</template>
+
+<script setup>
+import { DtEmoji } from '@/components/emoji';
+import { emojiJson } from '@/common/emoji';
+
+// TO DO EMOJI JSON = EMOJI LIST AND UTILITY FUNCTIONS
+
+import { computed } from 'vue';
+const props = defineProps({
+  /**
+   * The filter to apply to the emoji list
+   * @type {String}
+   * @default ''
+   */
+  emojiFilter: {
+    type: String,
+    default: '',
+  },
+
+  /**
+   * The skin tone to apply to the emoji list
+   * @type {String}
+   * @default null
+   */
+  skinTone: {
+    type: Number,
+    default: null,
+  },
+});
+
+const emits = defineEmits([
+  /**
+   * Emitted when the user hover over an emoji
+   * @event emoji-data
+   * @param {Object} emoji - The emoji data that was hovered
+    */
+  'emoji-data',
+
+  /**
+   * Emitted when the user select an emoji
+   * @event selected-emoji
+   * @param {Object} emoji - The emoji data that was selected
+    */
+  'selected-emoji',
+]);
+
+// emojiJson
+const emojiList = [];
+
+/**
+ * Filtered emoji list based on the filter
+ */
+const filteredEmojiList = computed(() => {
+  if (props.emojiFilter) {
+    return emojiList.filter((emoji) => emoji.includes(props.emojiFilter));
+  }
+  return emojiList;
+});
```

Comment:
We're also going to need to filter keywords

Reply:
Yes wil add a TODO in the code to not forget 👍🏼